tools/ncf.py

```diff
@@ -201,6 +206,17 @@ def parse_bundlefile_metadata(content, bundle_type):
 
   # If we found any parameters, store them in the res object
   if len(parameters) > 0:
+    for param in parameters:
+      constraints = {}
+      if len(param_constraints) > 0:
+        constraint_array = [ constraint["constraint"] for constraint in param_constraints if constraint["name"] == param["name"] ]  
```

Comment:
I think a warning is a good idea
At least that should not be an 'error' that prevents reading the technique